rock-paper-scissors/script.js

```diff
@@ -0,0 +1,31 @@
+function computerPlay(){
+	const options = ['rock', 'paper', 'scissors'];
+	return options[Math.floor(Math.random()*options.length)];
+}
+ 
+function playRound(playerSelection, computerSelection){
+	if (playerSelection === computerSelection){
+		console.log(`You and the computer played ${playerSelection}. It's a tie`);
+	}
+	else if ((playerSelection === 'rock' && computerSelection === 'scissors')
+			||(playerSelection === 'scissors' && computerSelection === 'paper')
+			 ||(playerSelection === 'paper' && computerSelection === 'rock')){
+		console.log(`Computer played ${computerSelection}. You win!`);
+			 }
+	else
+		console.log(`Computer played ${computerSelection}. You lose.`);
+}	
+ 
+ 
+ 
+function game(){
+	for (let i = 0; i < 5; i++){
+		do {
+			playerSelection = prompt('rock, paper or scissors','')
+}
+		while (playerSelection.toLowerCase() !== 'rock' && playerSelection.toLowerCase() !== 'paper' && playerSelection.toLowerCase() !== 'scissors');
```

Comment:
I'd suggest not using a do-while loop, just bc it's not super common, but up to you - it's definitely more personal preference. I'd suggest a `while (true) { ... }` and then put an `if (...) break;` inside. 

Also, can you add 2 methods? 
1. a method to validate the player input ("is this a valid string") that returns `true` or `false`
2. a method called `playerPlay()` or something opposite to `computerPlay()` that retrieves player input